config/roguelike_dungeons/settings/loot_armor.json

```diff
@@ -0,0 +1,126 @@
+		{"data": {"name": "ExtraUtilities:glove", "meta": 0}, "weight": 1}
+
+
+      ]
+    },
+	{"level": 0, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "BiomesOPlenty:bootsMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:chestplateMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:helmetMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "BiomesOPlenty:leggingsMud", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:helmetWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:leggingsWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:chestplateWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:bootsWood", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_boots", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_chestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_helmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:leather_leggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:slime_boots", "meta": 0}, "weight": 4},
+		{"data": {"name": "battlegear2:shield.wood", "meta": 0}, "weight": 15}
+
+      ]
+    },
+    {"level": 1, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "harvestcraft:hardenedleatherbootsItem", "meta": 0}, "weight": 13},
+		{"data": {"name": "harvestcraft:hardenedleatherchestItem", "meta": 0}, "weight": 13},
+		{"data": {"name": "harvestcraft:hardenedleatherhelmItem", "meta": 0}, "weight": 13},
+		{"data": {"name": "harvestcraft:hardenedleatherleggingsItem", "meta": 0}, "weight": 13},
+		{"data": {"name": "harvestthenether:quartzbootsItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzchestItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzhelmItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzleggingsItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "adventurebackpack:adventureHat", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:adventurePants", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:adventureSuit", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:pistonBoots", "meta": 0}, "weight": 5},
+		{"data": {"name": "minecraft:gold_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.hide", "meta": 0}, "weight": 15}
```

Comment:
battlegear2 is going to be removed, give out a fully built Tinker's Defense shield here instead.